lib/xenops_server.ml

```diff
@@ -104,7 +104,7 @@ type atomic =
   | VM_create_device_model of (Vm.id * bool)
   | VM_destroy_device_model of Vm.id
   | VM_destroy of Vm.id
-  | VM_create of (Vm.id * int64 option * Vm.id option)
+  | VM_create of (Vm.id * int64 option * Vm.id option * bool (*sriov*))
```

Comment:
why not alias the bool with `type is_sriov = bool` to avoid the need for this comment? i noticed a similar one further down as well

Reply:
It's a good idea. I think the I should add a comment what this flag actually does.